docker-compose/jaeger-docker-compose.yml

```diff
@@ -1,11 +1,32 @@
 version: '2'
 
 services:
+    hotrod:
+      image: jaegertracing/example-hotrod:latest
```

Comment:
if this docker-compose/jaeger-docker-compose.yml is used in any crossdock tests then including hotrod might create conflicts in the ports (but might be ok if host names are used for addressing)

Reply:
`docker-compose/jaeger-docker-compose.yml` does not seem to be referenced by anything else in the repo. I figured it was a dev/debugging tool so it seemed ok to make updates.

Reply:
We used to import it (https://github.com/jaegertracing/jaeger-client-java/pull/276/files) but then removed in favor of only running all-in-one (https://github.com/jaegertracing/jaeger-client-java/pull/654/files)

Reply:
Let's add a header to this file explaining why it exists, where it can be / should not be used.